README.md

```diff
@@ -3,12 +3,30 @@
 This repo forks the FreeRDP repo, thus allowing us to make changes that fit our needs and augment the codebase with other components.
 From time to time there is a need to merge the changes from the original repo into this one.
 
-### ❗When updating the FreeRDP library from the official repo, please update the following in this file (README.md):
+### ❗When updating the FreeRDP library from the official repo, please update the following table in this file (README.md):
 
-|**Original repository tag/branch used:**| `2.5.0` |
+|**Original repository tag/branch used:**| `2.11.2` |
 | --- | --- |
-|**Original corresponding commit hash:**| `d50aef95520df4216c638495a6049125c00742cb` |
+|**Original corresponding commit hash:**| `a38c1be9eee39a9bc22b511fffe96e63fdf8ebe7` |
 
+#### Updating the fork is done on a dev machine and involves rebasing our changes on top of the new tag/branch from the original repo and **force pushing** the `uipath` branch.
+
+❗The state before the update must be saved in a support branch, named `robot/support/before_update_to_<NEW BASE GIT TAG>`, where `<NEW BASE GIT TAG>` is the tag from the original repo onto which we'll rebase our changes.
+
+The tags from the original repo must be fetched from the original repo and pushed to our fork:
+```pwsh
+git fetch --tags --all
+git push --tags
+```
+
+A work branch should be created from `uipath` and used in the validation phase. It's name would be `feat/update_to_<NEW BASE GIT TAG>`.
```

Comment:
@daniel-stanciu, can you comment?

I feel it's irrelevant if the work branch was created from the `uipath` branch or not. It could point to any commit. However, if it's not identical to `uipath`, then the `git rebase --onto` command will need to mention `uipath` as it's 4th argument, instead of the work branch.